libraries/abstractions/pkcs11/utest/iot_pkcs11_mbedtls_utest.c

```diff
@@ -36,18 +37,90 @@
 #include "mock_queue.h"
 #include "mock_ctr_drbg.h"
 #include "mock_entropy.h"
+#include "mock_sha256.h"
+#include "mock_pk.h"
+#include "mock_ecp.h"
+#include "mock_rsa.h"
+#include "mock_bignum.h"
+#include "mock_portable.h"
 
 /* PKCS #11 includes. */
 #include "iot_pkcs11_config.h"
 #include "iot_pkcs11.h"
+/* This mock must be included after pkcs11.h */
+#include "mock_iot_pkcs11_pal.h"
+
+/* mbedtls includes. */
+#include "oid.h"
 
 /* Test includes. */
 #include "unity.h"
 
 
-
+/* ============================  GLOBAL VARIABLES =========================== */
+/* Length parameters for importing EC private keys. */
+#define EC_PARAMS_LENGTH    10
+#define EC_D_LENGTH         32
+
+#define EC_PRIV_KEY_INITIALIZER                                                    \
+    {                                                                              \
+        { CKA_CLASS, &xPrivateKeyClass, sizeof( CK_OBJECT_CLASS ) },               \
+        { CKA_KEY_TYPE, &xPrivateKeyType, sizeof( CK_KEY_TYPE ) },                 \
+        { CKA_LABEL, pucLabel, ( CK_ULONG ) strlen( ( const char * ) pucLabel ) }, \
```

Comment:
If this is used with a variable declaration, then I think that the strlen causes a compiler warning. If so, can you please refactor?

Reply:
I compiled this on GCC with -Wall -Wpedantic -Wextra and confirmed no warnings  👍